### DISCUSSION TITLE
Documentar canales y sus especificaciones

### DISCUSSION THREAD

**Discussion by 5011-5011** [nextgenhealthcare/connect#5330] [Q&A] [answered]
¿ Podria alguien orientarme como armar documentacion acerca de los canales? Comence hace un par de meses a utilizar esta herramienta, y tengo varios canales implementados, pero debo documentales y no encuentro alguna referencia. **Tengo en mente la info que debo incluir pero quisiera saber si hay algun prototipo estandar para realizar esta documentacion.**

**Comment by pacmano1** [comment, marked as answer]
Nextgen tenía una herramienta para pagar a los clientes.  Hoy en día, no creo que se utilice un prototipo estándar.
